data/resources.json

```diff
@@ -8,10 +8,12 @@
   "followOur": "Follow our",
   "followUs": "Follow us",
   "emailUs": "Email us",
-  "blog": "Blog",
   "partnershipResources": "Partnership Resources",
   "eventResources": "Event Resources",
   "buyTicket": "Buy ticket",
+  "footer": {
+    "blog": "Blog"
```

Comment:
Haven't you used this for the toolbar? It can be common and used in different places